packages/pwa-devdocs/src/_data/peregrine.yml

```diff
@@ -18,5 +18,8 @@ entries:
       - label: List
         url: /peregrine/reference/list/
 
+      - label: REST API client
+        url: /peregrine/reference/rest-api-client/
+
```

Comment:
In Peregrine, a `Component` isn't a generic term like it is for other contexts. Peregrine includes utilities and stuff, but its Components are literally [React Components](https://reactjs.org/docs/react-component.html). So I don't think we should list the REST API client in Components, since it's just a JavaScript module and not a React component.

I think this might demand a separate section for "Modules", with short explanations that Components are React Components and Modules are other utilities.